pynetgear/__init__.py

```diff
@@ -271,6 +272,71 @@ def tofloats(lst): return (float(t) for t in lst)
             return None
 
         return {t.tag: parse_text(t.text) for t in node}
+        
+    def config_start(self):
+        """
+        Start a configuration session.
+        For managing router admin functionality (ie allowing/blocking devices)
+        """
+        _LOGGER.info("Config start")
+
+        message = "<NewSessionID>{session_id}</NewSessionID>".format(session_id=SESSION_ID)
+
+        success, _ = self._make_request(
+            SERVICE_DEVICE_CONFIG, "ConfigurationStarted", message)
```

Comment:
_make_request can take a dict as params that will be inserted as a body. Here for example you can directly pass `{"NewSessionID": SESSION_ID}` like here:
https://github.com/MatMaul/pynetgear/blob/master/pynetgear/__init__.py#L93

This applies everywhere applicable :)

Reply:
Fixed :)